community/sdl2_net/build

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh -e
+
+./configure --prefix=/usr
```

Comment:
It is the preferred style to split this:
```sh
./configure \
    --prefix=/usr
````
https://kisslinux.org/wiki/package-system#001

Reply:
Thanks!